client/components/chart/index.jsx

```diff
@@ -99,6 +103,10 @@ class Chart extends React.Component {
 		} );
 	};
 
+	setTooltip = ( tooltipContext, tooltipPosition, tooltipData ) => {
```

Comment:
I'd name this function `showTooltip` and give it a better API: setting `tooltipContext` to `null` and the other two to `undefined` looks sloppy. Also, a separate `tooltipVisible` wouldn't be bad.

Reply:
We could but I don't think it would make a difference.

Reply:
fwiw i was able to grok what was going on here by reading the PR so it seems okay to me. But yes it would work well either way.